backend/dataall/cdkproxy/stacks/policies/service_policy.py

```diff
@@ -106,27 +93,9 @@ def generate_policies(self) -> [aws_iam.ManagedPolicy]:
 
         services = ServicePolicy.__subclasses__()
 
-        if permissions.CREATE_REDSHIFT_CLUSTER not in self.permissions:
```

Comment:
[clarification] - the check of the permissions needed is moved to the modules (I really like this). @nikpodsh, I am thinking that for some modules we will need permissions form different services. e.g. It is not the same the permissions needed for Notebook than for SM Studio (although both are SageMaker), right now we are taking the sum of both for both modules, if instead we defined the permissions not by service, but by module we would have less inter-dependency and we would be able to restrict permissions to only those needed for each module. (Of course it has the downside of duplicating permissions, but I think in this case security comes always first)

Reply:
Yes, the permission should be defined by the module in the most cases. The example can be found in `modules/notebooks/services/permissions.py. 
But here we have a special case: notebooks and ML studio share the permissions in a few places. It was the reason why I put a few permissions in common. 

The right solution would be to define more granular permissions for each module, but I had had to write a migration script for existing permissions and wanted to avoided this overhead to keep the PR smaller 🙂

Reply:
I agree here with @dlpzx - we should keep the permissions inside the module. These are part of the business logic that is encapsulated inside module

Reply:
- [x] Migrated permissions. Now, ML studios have they own permissions and there are no interaction with notebook permissions